graphitepager/graphite_target.py

```diff
@@ -23,7 +23,7 @@ def get_records(base_url,
 
 
 def _graphite_url_for_target(base, target, from_='-1min', until_=None):
-    url = '{0}/render/?target={1}&rawData=true&from={2}'.format(
+    url = '{0}/render/?target={1}&format=raw&from={2}'.format(
```

Comment:
`rawData=true` was deprecated in Graphite 0.9.9 so I went ahead and updated while I was in here.

Ref: https://graphite.readthedocs.io/en/latest/render_api.html#rawdata